jastrow.py

```diff
@@ -1,20 +1,44 @@
 import numpy as np
 from func3d import GaussianFunction
 
-
-def eedist(configs):
+def eedist_old(configs):
```

Comment:
Instead of naming things 'old' and new, be descriptive. What is different about these different functions?

Reply:
like  eedist and  eedist_spin. also document the inputs/outputs

Reply:
Changed function names to eedist/eidist and eedist_spin/eidist_spin and updated docstring for spin cases.